wire-java-generator/src/main/java/com/squareup/wire/java/JavaGenerator.java

```diff
@@ -825,7 +892,7 @@ private TypeName fieldType(Field field) {
           typeName(type.valueType()));
     }
     TypeName messageType = typeName(type);
-    return field.isRepeated() ? listOf(messageType) : messageType;
+    return field.isRepeated() || field.isPacked() ? listOf(messageType) : messageType;
```

Comment:
I’m curious . . . do we ever have fields that are packed but not repeated? That’s a horrible bug that we should fix. I don’t like this change.